psalm-report.json

```diff
@@ -1,5 +1,6944 @@
 {
-    "error": [],
-    "warning": [],
-    "deprecation": []
+    "error": [
+        {
+            "severity": "error",
+            "line_from": 54,
+            "line_to": 54,
+            "type": "ParamNameMismatch",
+            "message": "Argument 1 of SprykerEco\\Yves\\Computop\\Converter\\AbstractInitConverter::getResponseTransfer has wrong name $responseHeader, expecting $decryptedArray as defined by SprykerEco\\Yves\\Computop\\Converter\\ConverterInterface::getResponseTransfer",
+            "file_name": "vendor/spryker-eco/computop/src/SprykerEco/Yves/Computop/Converter/AbstractInitConverter.php",
+            "snippet": "    public function getResponseTransfer(array $responseHeader): TransferInterface",
+            "selected_text": "$responseHeader",
+            "error_level": 7,
+            "shortcode": 230
+        },
+        {
+            "severity": "error",
+            "line_from": 24,
+            "line_to": 24,
+            "type": "ParamNameMismatch",
+            "message": "Argument 1 of SprykerEco\\Yves\\Computop\\Converter\\InitPayuCeeSingleConverter::createResponseTransfer has wrong name $decryptedArray, expecting $responseParamsArray as defined by SprykerEco\\Yves\\Computop\\Converter\\AbstractInitConverter::createResponseTransfer",
+            "file_name": "vendor/spryker-eco/computop/src/SprykerEco/Yves/Computop/Converter/InitPayuCeeSingleConverter.php",
+            "snippet": "    protected function createResponseTransfer(array $decryptedArray, ComputopApiResponseHeaderTransfer $header): TransferInterface",
+            "selected_text": "$decryptedArray",
+            "error_level": 7,
+            "shortcode": 230
+        },
+        {
+            "severity": "error",
+            "line_from": 119,
+            "line_to": 119,
+            "type": "UndefinedMethod",
+            "message": "Method Generated\\Shared\\Transfer\\ComputopEasyCreditPaymentTransfer::setPackingStation does not exist",
```

Comment:
This should not be here if you generated the transfers as per test and phpstan need and also run then psalm with those generated.

Reply:
Fixed